code-challenges.js

```diff
@@ -45,10 +106,79 @@ var stringWithVowels2 = "ILoveJavaScript"
 
 
 
-
-// --------------------6) Expand the logic from #4 to inform your user if the variable passed into the vowel removal function is not a string.
+//create a function
+const noVowels = (string) => {
+    //created a new variable no store our string with removed vowels.
+    
+    //string.replace we enter what we want to be replaced in between two /'s. It can be a word or single characters separated by |, in this case we used | to separate vowels. 
+    
+    //The argument following the / / can either be g (global), or gi (global case-insensitive). Used gi to cover uppercase vowels as well.
+    
+    //Then it is followed by what we want it replaced with. In this case nothing so we put ""
+    let nVwls = string.replace(/a|e|i|o|u/gi, "")
```

Comment:
Oooh, REGEX! Fancy! 

As Rachael mentioned in review, since you were able to explain whats going on with your great pseudo coding and how to use this- this is totally cool!  If you find new ways to do things, just be sure to always pseudo code it out how it works and how you will use it. 

Good work!